code/go/0chain.net/smartcontract/storagesc/free_allocation.go

```diff
@@ -35,8 +35,9 @@ func (frm *freeStorageMarker) decode(b []byte) error {
 }
 
 type freeStorageAllocationInput struct {
-	RecipientPublicKey string `json:"recipient_public_key"`
-	Marker             string `json:"marker"`
+	RecipientPublicKey string   `json:"recipient_public_key"`
+	Marker             string   `json:"marker"`
+	PreferredBlobbers  []string `json:"preferred_blobbers"`
```

Comment:
Rename `PrefferedBlobbers` to `Blobbers` if this API is new, otherwise we need to update gosdk to adjust the change of API input.